src/app/login/login.page.html

```diff
@@ -0,0 +1,61 @@
+<ion-content>
+  <div class="background"><div class="background-inner"></div></div>
+  <div class="header">
+    <h1 class="max-wrap ion-padding">
+      Volunteer Login
+    </h1>
+  </div>
+
+  <div class="max-wrap">
+    <ion-card>
+      <ion-card-content>
+        <form #form="ngForm" (ngSubmit)="signin(form)">
```

Comment:
it can't be called `#form` because we also have a ngModel called `form`. I'd call it anything else other than `form` but I would recommend `ngForm`

Reply:
(this will also make your validation messages work!)